### PR TITLE
72 auth pages tasque input

### PR DESCRIPTION
**Pull request by ihor-il**
__Migrated auth forms from default input to `tasque-input`__
- [x] visuals
- [x] functionality

### CODE REVIEW

File: frontend/src/app/auth/components/login-page/login-page.component.ts [hunk 1]
```diff
@@ -77,13 +107,29 @@ export class LoginPageComponent implements OnInit, OnDestroy {
...
+      .loginUser(this.userLogin)
+      .pipe(takeUntil(this.unsubscribe$))
+      .subscribe((resp) => {
+        if (resp.ok) {
+          const token = resp.body;
+          this.localStorage.setItem(
+            this.localStorageKeys.token,
+            token?.accessToken as string,
+          );
+        }
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/auth/components/login-page/login-page.component.ts]: No localstorage moves in component, move it to authService or another one

File: frontend/src/app/auth/components/restore-page/restore-page.component.html [hunk 2]
```diff
@@ -19,10 +19,8 @@ <h4 class="login-title">Forgot password</h4>
         />
         <span class="error" *ngIf="emailControl.errors?.['required'] && (emailControl.dirty || emailControl.touched)">Email is required.</span>
         <span class="error" *ngIf="emailControl.errors?.['pattern']">Incorrect email format</span>
-      </div>
-    </form>
-  </div>
+      </div> -->
```
Comment (Anatolii-Ivanov) [inline frontend/src/app/auth/components/restore-page/restore-page.component.html]: I think you can drop it

File: frontend/src/app/auth/components/login-page/login-page.component.ts (hunk 1 above)
Comment (ihor-il) [reply frontend/src/app/auth/components/login-page/login-page.component.ts]: Done

File: frontend/src/app/auth/components/reset-page/reset-page.component.html [hunk 3]
```diff
@@ -1,65 +1,23 @@
...
-    <div>
-      <form class="login-form">
+    <div class="form-container">
+      <div class="text-container">
         <h4 class="login-title">Reset password</h4>
-        <div class="text-container">
-          <span>Sorry, your password reset key is invalid. Request new email or contact technical support </span>
-        </div>
-      </form>
+        <span class="label-14">Sorry, your password reset key is invalid. Request new email or contact technical support </span>
```
Comment (Its-OP) [inline frontend/src/app/auth/components/reset-page/reset-page.component.html:19]: > Request new email

Request new message

**Review by Its-OP — CHANGES_REQUESTED** [review]

File: frontend/src/app/auth/components/reset-page/reset-page.component.html (hunk 3 above)
Comment (ihor-il) [reply frontend/src/app/auth/components/reset-page/reset-page.component.html:19]: Maybe `Request new link`?

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by Vadis00 — APPROVED** [review]

**Review by Its-OP — APPROVED** [review]

**Pull request merged**